lib/commands/help.js

```diff
@@ -61,13 +63,27 @@ class Help extends BaseCommand {
     const f = `${manroot}/${manSearch}/?(npm-)${section}.[0-9]*`
     let mans = await glob(f)
     mans = mans.sort((a, b) => {
-      // Because of the glob we know the manNumberRegex will pass
-      const aManNumber = a.match(manNumberRegex)[1]
-      const bManNumber = b.match(manNumberRegex)[1]
+      // Prefer the page with an npm prefix, if there's only one.
+      const aHasPrefix = a.match(manNpmPrefixRegex)
+      const bHasPrefix = b.match(manNpmPrefixRegex)
+      if (!aHasPrefix !== !bHasPrefix) {
```

Comment:
this seems like a really weird way to do this; if you’re not using the match object, do regex.test(str) instead?